test/functional/feature_rbf.py

```diff
@@ -42,6 +42,10 @@ def set_test_params(self):
                 "-limitdescendantcount=200",
                 "-limitdescendantsize=101",
             ],
+            # second node has default mempool parameters
+            [
+                "-acceptnonstdtxn=1",
```

Comment:
Not sure if this is me being pedantic, but I don't think we're really testing "default mempool parameters" if we're configuring the node to `acceptnonstdtxn`. Is there a good reason we need to set this?

Reply:
I think the only reason this needs to accept nonstd txs is because there's some left over txs in node 0's mempool from earlier tests, which results in syncing mempools failing when you try to mine a block with node 1. Doing:

```diff
+        #clear the mempools
+        self.generate(self.nodes[0], 1)
+
         normal_node = self.nodes[1]
         wallet = MiniWallet(normal_node)
         wallet.rescan_utxos()            
```

Looks to me like it lets you drop the `-acceptnonstdtxn=1`.